push/parser.go

```diff
@@ -284,7 +311,10 @@ type Update interface {
 // BaseUpdate contains fields & methods related to update-based messages
 type BaseUpdate struct {
 	BaseMessage
-	changeNumber int64
+	changeNumber         int64
+	previousChangeNumber int64
```

Comment:
this is not BaseUpdate struct, it should be `type SplitChangeUpdate struct`

Reply:
I moved them.